solutions/swb-reference/integration-tests/support/complex/dataSetHelper.ts

```diff
@@ -0,0 +1,52 @@
+import { AccessPoint } from '@aws-sdk/client-s3-control';
+import { AwsService } from "@aws/workbench-core-base";
+import _ from "lodash";
+import ClientSession from "../clientSession";
+
+export class DataSetHelper {
```

Comment:
Should we call this `DatasetHelper` to keep the naming consistent? I notice in other class `Set` is lowercase